app/controllers/people_controller.rb

```diff
@@ -47,6 +47,9 @@ def destroy
   private
 
   def person_params
-    params.require(:person).permit(:first_name, :last_name, :email, :password, :password_confirmation, :date_of_birth, :gender)
+    params.require(:person).permit(
+      :first_name, :last_name, :email, :password,
+      :password_confirmation, :date_of_birth, :gender
```

Comment:
The advantage of listing one arg per line is that if a single param changes in the future the git diff will be much easier to read. 

if there are multiple things on one line, and one item in the list changes, git highlights the whole line making it harder to identify what exactly on that line changed.

Ultimately, this is the kind of thing that we should rely on rubocop for. So long as we agree on which style we prefer, we can set up rubocop to enforce it